ui/src/components/UI/StatusCode.tsx

```diff
@@ -7,24 +7,27 @@ export enum StatusCodeClassification {
     NEUTRAL = "neutral"
 }
 
-
-interface HAREntryProps {
+interface EntryProps {
     statusCode: number
 }
 
-const StatusCode: React.FC<HAREntryProps> = ({statusCode}) => {
+const StatusCode: React.FC<EntryProps> = ({statusCode}) => {
 
     const classification = getClassification(statusCode)
 
-    return <span className={`${styles[classification]} ${styles.base}`}>{statusCode}</span>
+    return <span
+        title="Status Code"
+        className={`${styles[classification]} ${styles.base}`}>
+            {statusCode}
+    </span>
 };
 
 export function getClassification(statusCode: number): string {
     let classification = StatusCodeClassification.NEUTRAL;
 
-    if (statusCode >= 200 && statusCode <= 399) {
+    if ((statusCode >= 200 && statusCode <= 399) || statusCode === 0) {
```

Comment:
what does it mean statusCode 0-16?
Maybe let's add a comment here to explain this